setup.py

```diff
@@ -261,6 +235,10 @@ def gen_packages_items():
                 name='nms_cuda',
                 module='mmdet.ops.nms',
                 sources=['src/nms_cuda.cpp', 'src/nms_kernel.cu']),
+            make_cuda_ext(
+                name='soft_nms_cpu',
+                module='mmdet.ops.nms',
+                sources=['src/soft_nms_cpu.cpp']),
```

Comment:
I suggest to use only one extension for NMS and SoftNMS.

Reply:
I will merge them into one cpp file.